serviceWrappers/linux/kLiveController

```diff
@@ -32,18 +37,17 @@ HOME=${HOME:-~}
 
 loadNvm() {
     # This loads nvm
-    cd $KLIVE_CONTROLLER_PREFIX
-    [ -z "$NVM_DIR" ] && NVM_DIR="$HOME/.nvm"
-    if [ -s "$NVM_DIR/nvm.sh" ] ; then
-        source "$NVM_DIR/nvm.sh"
-        if [ -s .nvmrc ] ; then
+    [ -z "${NVM_DIR}" ] && NVM_DIR="${HOME}/.nvm"
+    if [ -s "${NVM_DIR}/nvm.sh" ] ; then
+        source "${NVM_DIR}/nvm.sh"
+        if [ -s ${KLIVE_CONTROLLER_PREFIX}/.nvmrc ] ; then
             nvm use || nvm install
         else
-            echo "no .nvmrc in $PWD , Exiting"
+            echo "no .nvmrc in ${PWD} , Exiting"
```

Comment:
Change this to no " ${KLIVE_CONTROLLER_PREFIX}/.nvmr", CWD should never be relied upon.